### DISCUSSION TITLE
Option to disable webrtc in chrome?

### DISCUSSION THREAD

**Discussion by User3409** [kkapsner/CanvasBlocker#678] [Ideas]
In chrome, I am surprised the fingerprinting of using webrtc to resolve local IP address has never been fixed.
I read extensions can disable it somehow. Should canvasblocker have an option to disable webrtc on chrome?

**Comment by kkapsner** [comment]
There are no plans to provide CB for Chrome (#65). If you have a scenario for a webRTC leakage please post it to #511.
